.github/dependabot.yml

```diff
@@ -10,3 +10,15 @@ updates:
       github-actions:
         patterns:
           - "*"
+  - package-ecosystem: "gitsubmodule"
```

Comment:
We might want to include the date in the commit message and PR title, but I don't see a way to do that: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#commit-message--

(I'm seeing how consistent with https://github.com/ROCm/TheRock/pull/2401)